nuxeo-platform-video-convert/src/test/java/org/nuxeo/ecm/platform/video/convert/VideoConversionTest.java

```diff
@@ -99,7 +95,7 @@ protected BlobHolder applyConverter(String converter, String fileName, String mi
 
     @Test
     public void testWebMConversion() throws Exception {
-        CommandLineExecutorService cles = Framework.getLocalService(CommandLineExecutorService.class);
+        CommandLineExecutorService cles = Framework.getService(CommandLineExecutorService.class);
```

Comment:
While you're changing this, you could `@Inject` it as well instead of doing an explicit lookup. (And there's no need for the `assertNotNull(cles)` then.)